src/config/configOptionMigrations.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
     // Set DashPay contract ID and block height for testnet
     lodashSet(options, 'platform.dashpay', systemConfigs.testnet.platform.dashpay);
 
+    return options;
+  },
+  '0.18.0': (name, options) => {
+    lodashSet(options, 'core.sentinel.docker.image', 'dashpay/sentinel:1.5.0');
```

Comment:
It won’t work ‘coz you don’t have sentinel object. So you need to set `systemConfigs.base.core.sentinel` with `core.sentinel` key. It’s better to test before :))

Reply:
I thought I can't test this unless it is released? I still don't understand how this function works, so I am literally guessing in these commits. Please show me how to test it.

Reply:
Just set 0.18.0 in package json and run any command. Please don’t forget to bring version back before you commit.